synapseclient/client.py

```diff
@@ -3591,18 +3591,16 @@ def sendMessage(self, userIds, messageSubject, messageBody, contentType="text/pl
     #                   Low level Rest calls                   #
     ############################################################
 
-    def _generateSignedHeaders(self, url, headers=None):
+    def _generateHeaders(self, url, headers=None):
```

Comment:
Any reason for the camel case?  If the function is being renamed, should we just do `_generate_headers`

Reply:
Good point.